src/Symfony/Component/DependencyInjection/Loader/YamlFileLoader.php

```diff
@@ -775,6 +775,10 @@ private function resolveServices($value, $file, $isParameter = false)
                 $value[$k] = $this->resolveServices($v, $file, $isParameter);
             }
         } elseif (is_string($value) && 0 === strpos($value, '@=')) {
+            if (!class_exists(Expression::class)) {
+                throw new \LogicException(sprintf('The @= expression syntax cannot be used without the expression component. Try running "composer require symfony/expression-language".'));
```

Comment:
`The "@=" expression syntax cannot be used without the ExpressionLanguage component. Try running "composer require symfony/expression-language".`

Reply:
(reminder: strings in double quotes are in bold in profiler panels ;) )